packages/react-query/src/internals/context.tsx

```diff
@@ -240,5 +241,18 @@ export interface TRPCQueryUtils<TRouter extends AnyRouter> {
   getInfiniteQueryData: (
     queryKey: TRPCQueryKey,
   ) => InfiniteData<unknown> | undefined;
+
+  /**
+   * @link https://tanstack.com/query/latest/docs/reference/QueryClient/#queryclientsetmutationdefaults
+   */
+  setMutationDefaults: (
+    mutationKey: string[][],
```

Comment:
Not sure about this type. `string[][]` should be better than `MutationKey` from TanStack Query, which is defined as `ReadonlyArray<unknown>`. But I am not sure what the relationship between this type and `TRPCQueryKey` should be, assuming there should be a relationship

Reply:
Probably `[string[]]` is a better choice here (= `TRPCQueryKey` without the optional second value of the tuple)

We can also define it as `[TRPCQueryKey[0]]` or define a new alias

Reply:
Updated in https://github.com/trpc/trpc/pull/5814/commits/790ec8e0e541f651dc13abfd407976de12823a5d